apps/molgenis-components/src/components/utils/utils.js

```diff
@@ -0,0 +1,31 @@
+import constants from "../constants";
```

Comment:
could just move the utils file into the components folder ( or higher), not sure what adding the utils folders does